CONTRIBUTING.md

```diff
@@ -129,6 +129,21 @@ When you have set up your fork of a repository that you want to contribute to an
 4. **Validating and verifying the PR:** 
    - Reviewers can review the pull request on the GitHub website or fetch the PR using the _Fetch GitHub PR_ menu entry in the _Git Repositories_ view inside the Eclipse IDE.
    - Every PR is automatically verified to check that contributors have a [valid Eclipse Contributor Agreement (ECA)](#setting-up-your-eclipse-and-github-account).
+   ### What does a valid PR look like? (Checklist)
+   These are the basic criteria that need to be fulfilled for a PR to be considered valid. 
+   
+   **If you are a contributor, consider these points as prerequisites before your PR can be reviewed. Please ask for help if needed, either by making a comment on the PR or the [dev mailing list](#contact).**.
+
+   - All checks need to pass:
+     - If tests are broken then they should either be fixed or documented (create a new issue or mention an existing one).
+     - The code must compile.
+     - The versions of the affected plugins should be properly bumped.
+     - If there are other (obscure) errors in the checks, please find them (read the logs) and actively ask what is happening and how to fix them.
+   - The commit text should be well written (see [Commit Message Recommendations](#commit_message_recommendations)).
+   - If tests were added, they should:
```

Comment:
```suggestion
   - If tests were added, they should meeting the following criteria:
```

Reply:
Shouldn't it be "... meet the following criteria"?

... and then:
* They should run: [...]
* They should pass: [...]

?

Reply:
Yes.

Reply:
I went for my own suggestion (https://github.com/eclipse-platform/.github/pull/212#discussion_r1633711495) in 97c526c8776910a4d12125ab25a72d1e3e2eab8b. I hope it's fine.